metadata_tools/index_support.py

```
@@ -4,14 +4,15 @@
 import fortranformat as ff
 import fnmatch
 import warnings
-import hosts.pds3 as pds3
+import json
```

Flake8 (7.3.0)

[error] 'json' imported but unused

(F401)